### PR TITLE
Newsletter 11: Add KatharosTech on GitHub Sponsors

### PR DESCRIPTION
**Pull request by zicklag**
_Part of #172 ("Newsletter 11: Coordination/Tracking")_

### CODE REVIEW

File: content/posts/newsletter-011/index.md [hunk 1]
```diff
@@ -69,6 +69,17 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library & Tooling Updates
 
+### Katharos Technology on GitHub Sponsors
+
+[Katharos Technology][ktech], a startup game and technology development studio, has gone live on [GitHub Sponsors][ghs_ktech]. Donations will go towards the development of their [Arsenal] game engine and supporting Rust gamdev libraries and tools such as [GFX], and [WGPU]. They have also [published an update][arsenal_update] on their website outlining the current state and direction of the Arsenal game engine.
```
Comment (ozkriff) [inline content/posts/newsletter-011/index.md:74]:
```suggestion
[Katharos Technology][ktech], a startup game and technology development studio, has gone live on [GitHub Sponsors][ghs_ktech]. Donations will go towards the development of their [Arsenal] game engine and supporting Rust gamedev libraries and tools such as [GFX], and [WGPU]. They have also [published an update][arsenal_update] on their website outlining the current state and direction of the Arsenal game engine.
```

**Comment by zicklag** [comment]
Absolutely. I reworded it quite a bit focusing on the vision and plan for the vision, is this too wordy/long?

**Comment by ozkriff** [comment]
Nice! We're trying to avoid h4 headers in general: could you remove the subheaders, please?

<details>
  <summary>Like this</summary>

  image  
</details>

Also, could you resolve conflicts, please?

I'd do the formatting tweaks and conflict resolution myself but the "Allow edits from maintainers" checkbox in this PR seems to be unselected.

**Comment by zicklag** [comment]
> I'd do the formatting tweaks and conflict resolution myself but the "Allow edits from maintainers" checkbox in this PR seems to be unselected.

That's strange, the checkbox is normally checked by default, and now I don't even see it. Maybe it is because it is from my organization repo or something. Anyway I made the changes. :)

**Comment by ozkriff** [comment]
Merged, thanks for the PR! :)
